tests/unit/SubscriptionUtilsTest.ts

```diff
@@ -239,10 +268,143 @@ describe('SubscriptionUtils', () => {
 
             await Onyx.multiSet({
                 [ONYXKEYS.NVP_PRIVATE_OWNER_BILLING_GRACE_PERIOD_END]: getUnixTime(subDays(new Date(), 3)), // past due
-                [ONYXKEYS.NVP_PRIVATE_AMOUNT_OWNED]: 8010,
+                [ONYXKEYS.NVP_PRIVATE_AMOUNT_OWED]: 8010,
             });
 
             expect(SubscriptionUtils.shouldRestrictUserBillableActions(policyID)).toBeTruthy();
         });
     });
+
+    describe('getSubscriptionStatus', () => {
+        afterEach(() => {
+            // Onyx.clear();
```

Comment:
Is this comment required or we can remove?

Reply:
We can remove, sure